### PR TITLE
[release-tooling] Support input key as string

### PR DESCRIPTION
**Pull request by runtian-zhou**
### Description

Add support for inputting keys as string for executing governance proposal.

### Test Plan

Manually run the command locally.

### CODE REVIEW

File: aptos-move/aptos-release-builder/src/main.rs [hunk 1]
```diff
@@ -27,11 +29,29 @@ pub enum Commands {
...
         release_config: PathBuf,
         #[clap(short, long)]
-        test_dir: PathBuf,
-        #[clap(short, long)]
         endpoint: url::Url,
         #[clap(long)]
         framework_git_rev: Option<String>,
+        #[clap(subcommand)]
+        input_option: InputOptions,
+        #[clap(long)]
```
Comment (perryjrandall) [inline aptos-move/aptos-release-builder/src/main.rs:37]: set default to false

Comment (runtian-zhou) [reply aptos-move/aptos-release-builder/src/main.rs:37]: I think the clap will default to false but lemme add that.

File: aptos-move/aptos-release-builder/src/main.rs [hunk 2]
```diff
@@ -27,11 +29,29 @@ pub enum Commands {
...
+}
+
+#[derive(Subcommand, Debug)]
+pub enum InputOptions {
+    FromDirectory {
+        #[clap(short, long)]
+        test_dir: PathBuf,
+    },
+    FromArgs {
+        #[clap(long)]
```
Comment (perryjrandall) [inline aptos-move/aptos-release-builder/src/main.rs:49]: Can you maybe add quick comment / doc for these option?

**Review by perryjrandall — APPROVED** [review]
WAGMI

File: aptos-move/aptos-release-builder/src/main.rs [hunk 3]
```diff
@@ -51,21 +71,59 @@ async fn main() -> Result<()> {
...
+                        framework_git_rev: None,
+                        endpoint,
+                    }
+                },
+            };
 
             network_config.framework_git_rev = framework_git_rev;
 
-            aptos_release_builder::validate::validate_config(config, network_config).await
+            if mint_to_validator {
```
Comment (perryjrandall) [inline aptos-move/aptos-release-builder/src/main.rs:118]: I would even say, only allow this on local testnet, if someone try this on testnet / mainnet we should fail early and loudly

File: aptos-move/aptos-release-builder/src/main.rs [hunk 4]
```diff
@@ -51,21 +71,59 @@ async fn main() -> Result<()> {
...
+                network_config.mint_to_validator().await?;
+            }
+
+            network_config.set_fast_resolve(30).await?;
```
Comment (perryjrandall) [inline aptos-move/aptos-release-builder/src/main.rs:122]: does this actually exist on devnet? (Thinking about how we're gonna execute today)

File: aptos-move/aptos-release-builder/src/main.rs [hunk 5]
```diff
@@ -51,21 +71,59 @@ async fn main() -> Result<()> {
...
+            aptos_release_builder::validate::validate_config(config, network_config.clone())
+                .await?;
+            // Reset resolution time back to normal after resolution
+            network_config.set_fast_resolve(43200).await
```
Comment (perryjrandall) [inline aptos-move/aptos-release-builder/src/main.rs:126]: Can you store these constants somewhere

File: aptos-move/aptos-release-builder/src/main.rs (hunk 1 above)
Comment (runtian-zhou) [reply aptos-move/aptos-release-builder/src/main.rs:37]: Clap will set the flag to false if the argument is not provided. Lemme add a print statement just to let us know if we minted anything.

File: aptos-move/aptos-release-builder/src/main.rs (hunk 4 above)
Comment (runtian-zhou) [reply aptos-move/aptos-release-builder/src/main.rs:122]: This would exist.

File: aptos-move/aptos-release-builder/src/main.rs (hunk 2 above)
Comment (runtian-zhou) [reply aptos-move/aptos-release-builder/src/main.rs:54]: Comments added!

File: aptos-move/aptos-release-builder/src/main.rs (hunk 3 above)
Comment (runtian-zhou) [reply aptos-move/aptos-release-builder/src/main.rs:125]: Added the logic to verify with testnet/mainnet

File: aptos-move/aptos-release-builder/src/main.rs (hunk 5 above)
Comment (runtian-zhou) [reply aptos-move/aptos-release-builder/src/main.rs]: Done

File: aptos-move/aptos-release-builder/src/main.rs (hunk 1 above)
Comment (movekevin) [reply aptos-move/aptos-release-builder/src/main.rs:40]: Can we make it clear in the comment that this should only be used on a local network?

**Review by movekevin — APPROVED** [review]
Nice work!

**Pull request merged**